### PR TITLE
search tags across all shards

### PR DESCRIPTION
**Pull request by salmonthefish**
Initially search shows up to 5 per shard with a link to the regular qless tag search results.

### CODE REVIEW

**Comment by ANorwell** [comment]
awesome

File: lib/qless/sharded_ui/views/_client_jobs.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<% if client_jobs.empty? %>
+  <div class="page-header">
+    <h1>No Shards <small>What kind of operation are you running here?</small></h1>
```
Comment (james-lawrence) [inline lib/qless/sharded_ui/views/_client_jobs.erb:3]: +1

**Comment by james-lawrence** [comment]
no view tests?

File: lib/qless/sharded_ui.rb [hunk 2]
```diff
@@ -251,12 +251,57 @@ def json(obj)
       def sanitize_attr(attr)
         return attr.gsub(/[^a-zA-Z\:\_]/, '-')
       end
+
+      def strftime(t)
+        # From http://stackoverflow.com/questions/195740
```
Comment (james-lawrence) [inline lib/qless/sharded_ui.rb:256]: i wonder how many places have this code snippet randomly inserted....

**Comment by james-lawrence** [comment]
LGTM

**Pull request merged**